test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs

```diff
@@ -3,28 +3,49 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.IO;
+using System.IO.Packaging;
 using System.Threading.Tasks;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
 using NuGet.Packaging;
-using NuGet.Protocol;
 using NuGet.Test.Utility;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace NuGet.PackageManagement.UI.Test
 {
     public class PackageItemListViewModelTests : IClassFixture<LocalPackageSearchMetadataFixture>
     {
         private readonly LocalPackageSearchMetadataFixture _testData;
         private readonly PackageItemListViewModel _testInstance;
+        private static readonly ImageSource DefaultPackageIcon;
+        private readonly ITestOutputHelper output;
 
-        public PackageItemListViewModelTests(LocalPackageSearchMetadataFixture testData)
+        public PackageItemListViewModelTests(ITestOutputHelper output, LocalPackageSearchMetadataFixture testData)
         {
             _testData = testData;
             _testInstance = new PackageItemListViewModel()
             {
                 PackageReader = _testData.TestData.PackageReader,
             };
+            this.output = output;
```

Comment:
`_output = output;`

Avoid `this`.  See our [.editorconfig](https://github.com/NuGet/NuGet.Client/blob/dev/.editorconfig#L41-L45).  Do you see this warning in Error List?  Do you have the warnings hidden?

Reply:
fixed in next iteration. 
no, i don't see those warnings -- i don't think i've done anything to hide warnings. just double checked settings (like in tools/options/text editor/) from this article: https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019